app/views/interactive_pages/_edit_section.html.haml

```diff
@@ -0,0 +1,48 @@
+:ruby
+  if section_name == 'HEADER_BLOCK'
+    section_hidden_field = hidden_field_tag :section, InteractivePage::HEADER_BLOCK
+    selector_menu = embeddable_selector
+    section_embeddables_list = @page.header_block_visible_embeddables
+  elsif section_name == 'INTERACTIVE_BOX'
+    section_hidden_field = hidden_field_tag :section, InteractivePage::INTERACTIVE_BOX
+    selector_menu = embeddable_interactives_selector
+    section_embeddables_list = @page.interactive_box_visible_embeddables
```

Comment:
This and line 5 seem to only be getting the visible embeddables. I'd think during authoring it'd need to get all of the embeddables.